infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -108,12 +108,13 @@ public void bulkWrite(List<LogData> entries, long epoch) {
      * Trim addresses from log up to a prefix.
      *
      * @param address prefix address to trim to (inclusive)
+     * @param epoch   epoch stamped on the trim request.
      */
-    public void prefixTrim(@Nonnull Token address) {
+    public void prefixTrim(@Nonnull Token address, long epoch) {
```

Comment:
Can we make the variable a more expressive one such as prefixEpoch or similar. It might be confusing to maintainers considering the address also has epoch.

Reply:
Done.